sklearn/feature_extraction/dict_vectorizer.py

```diff
@@ -58,8 +58,13 @@ class DictVectorizer(BaseEstimator, TransformerMixin):
         Whether transform should produce scipy.sparse matrices.
         True by default.
     sort : boolean, optional.
-        Whether ``feature_names_`` and ``vocabulary_`` should be sorted when fitting.
+        Whether ``feature_names_`` and ``vocabulary_`` should be sorted when
+        fitting.
         True by default.
+    drop_first_categorical_level: boolean, optional
```

Comment:
I agree. I'm going to update the rest of the docstring to comply with your suggestion. I've noticed this is not consistent across the entire project, unfortunately, but the suggested style is more common.